### PR TITLE
add shrink_mds scenario to CI

### PR DESCRIPTION
**Pull request by rishabh-d-dave**
Add shrink_mds scenario to CI so that it can be tested against master.

### CODE REVIEW

File: ceph-ansible-pipeline/config/definitions/ceph-ansible-pipeline.yml [hunk 1]
```diff
@@ -613,6 +613,48 @@
...
+                  - name: 'ceph-ansible-prs-dev-centos-non_container-shrink_mds'
+                    current-parameters: true
+                  - name: 'ceph-ansible-prs-dev-centos-container-shrink_mds'
+                    current-parameters: true
+      - conditional-step:
+          condition-kind: shell
+          condition-command: |
+            #!/bin/bash
+            set -x
+            # if the target branch is master or stable-4.0 then we DON'T RUN these tests.
```
Comment (guits) [inline ceph-ansible-pipeline/config/definitions/ceph-ansible-pipeline.yml]:
```suggestion
            # if the target branch isn't stable-4.0 then we DON'T RUN these tests.
```

File: ceph-ansible-pipeline/config/definitions/ceph-ansible-pipeline.yml [hunk 2]
```diff
@@ -613,6 +613,48 @@
...
+            fi
+            git diff --name-only $(git show HEAD | grep Merge | head -n 1 | cut -d ':' -f2) | grep 'infrastructure-playbooks/shrink-mds'
+          on-evaluation-failure: dont-run
+          steps:
+            - multijob:
+                name: 'ceph-ansible shrink_mds playbook testing'
+                condition: SUCCESSFUL
+                execution-type: PARALLEL
+                projects:
+                  - name: 'ceph-ansible-prs-luminous-centos-non_container-shrink_mds'
```
Comment (guits) [inline ceph-ansible-pipeline/config/definitions/ceph-ansible-pipeline.yml]:
```suggestion
                  - name: 'ceph-ansible-prs-nautilus-centos-non_container-shrink_mds'
```

File: ceph-ansible-pipeline/config/definitions/ceph-ansible-pipeline.yml [hunk 3]
```diff
@@ -613,6 +613,48 @@
...
+                    current-parameters: true
+                  - name: 'ceph-ansible-prs-luminous-centos-container-shrink_mds'
```
Comment (guits) [inline ceph-ansible-pipeline/config/definitions/ceph-ansible-pipeline.yml]:
```suggestion
                  - name: 'ceph-ansible-prs-nautilus-centos-container-shrink_mds'
```

File: ceph-ansible-pipeline/config/definitions/ceph-ansible-pipeline.yml (hunk 1 above)
Comment (rishabh-d-dave) [reply ceph-ansible-pipeline/config/definitions/ceph-ansible-pipeline.yml]: Done.

File: ceph-ansible-pipeline/config/definitions/ceph-ansible-pipeline.yml (hunk 2 above)
Comment (rishabh-d-dave) [reply ceph-ansible-pipeline/config/definitions/ceph-ansible-pipeline.yml]: Done.

File: ceph-ansible-pipeline/config/definitions/ceph-ansible-pipeline.yml (hunk 3 above)
Comment (rishabh-d-dave) [reply ceph-ansible-pipeline/config/definitions/ceph-ansible-pipeline.yml]: Done.

**Pull request merged**